docs/graphql/manual/guides/integrations/graphcool-migration.rst

```diff
@@ -0,0 +1,115 @@
+
+Replace the password as required.
+
+Step 3: Connect to MySQL via mysql CLI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+   mysql --user=root --password=my-secret-pw --host=<host>
+
+Replace the host and password as necessary.
+
+Step 4: Create a database in MySQL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+   create database public;
+
+Step 5: Import data
+^^^^^^^^^^^^^^^^^^^
+
+Import graphcool's MySQL export into your local MySQL instance
+
+.. code-block:: bash
+
+   mysql --user=root --password=my-secret-pw --host=<host> public --binary-mode=1 < <pathtomysqlexport>
+
+Replace host and pathtomysqlexport as required. This step should have migrated schema with the data.
+
+6. Start Hasura and Postgres using docker-compose
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since MySQL is now setup with all the data from graph.cool, we need to create Hasura and Postgres instance to proceed.
+
+Refer the :ref:`Getting started <docker_simple>` guide to setup Hasura using ``docker-compose``.
+
+We will use ``pgloader`` to migrate from MySQL to Postgres. Refer their `installation guide <https://github.com/dimitri/pgloader>`__ for setting this up.
+
+Once you have installed, execute the following command:
+
+.. code-block:: bash
+
+   pgloader mysql://root:my-secret-pw@<host>/public postgresql://postgres:postgrespassword@<host>:5432/postgres
+
+Replace ``<host>`` as necessary.
+
+This step should have migrated data from MySQL to Postgres.
+
+Connect Hasura to Postgres
+--------------------------
+Once the dataset is in Postgres, Hasura should be able to track tables and relationships. 
+
+
+You might also have to look at :ref:`configuring enums differently <enums_graphql_engine>` in case you have them in your Graph.cool project. 
+
+Restructuring connection tables
+-------------------------------
+
+Now you can rename tables/columns to match your client side queries as required. 
+Do note that, for every one to one relationship, Graph.cool would have created a connection table to link them. This would require a bit of work to restructure. Currently there is no automation available for this step. Carefully review the connection tables and make the necessary changes.
+
+Migrating functions
+-------------------
+
+In case you have functions in Graph.cool, Hasura has an equivalent feature called :ref:`event triggers <event_triggers>`. Migrating this would involve taking your code and deploying it on a different platform (preferably serverless functions).
+
+Do note that for event triggers, the payload that Hasura sends might be different and you might have to change the way request body parameters are handled in your function code.
+
+
+Migrating auth
+--------------
+
+If you were using Auth0 with Graph.cool, the migration should be fairly straightforward. You can configure Hasura with Auth0 easily by following :ref:`this guide <guides_auth0_jwt>`.
```

Comment:
We should probably say something about other auth methods?

Reply:
Added other auth method instructions